packages/block-editor/src/components/inserter/block-patterns-tab/patterns-filter.js

```diff
@@ -50,10 +69,9 @@ export function PatternsFilter( {
 		currentPatternSourceFilter
 	);
 
-	// We also need to disable the directory and theme source filter options if the category
-	// is `myPatterns` otherwise applying them will also just result in no patterns being shown.
-	const shouldDisableNonUserSources =
-		getShouldDisableNonUserSources( category );
+	// We also hide the directory and theme source filter if the category is `myPatterns`
+	// otherwise there will only be one option available.
+	const shouldHideSourcesFilter = getShouldHideNonUserSources( category );
```

Comment:
Nit: The function name makes it sound like it only hides some of the options

```suggestion
	const shouldHideSourcesFilter = getShouldHideSourcesFilter( category );
```

Reply:
Oops I missed that 😆